libmscore/tempotext.cpp

```diff
@@ -236,7 +236,7 @@ QVariant TempoText::propertyDefault(P_ID id) const
       {
       switch(id) {
             case P_ID::TEMPO:             return 2.0;
-            case P_ID::TEMPO_FOLLOW_TEXT: return false;
+            case P_ID::TEMPO_FOLLOW_TEXT: return true;
```

Comment:
For properties, the default behavior is to not write them to MSCZ files if they aren't default. Apparently for this one, we don't use writeProperty and we will write it only if "true". Then, when we read, it's set to false in the constructor, and read from the file to be set to true if necessary.

If we change the default to true here, we loose the opportunity to call write/read property for this property :-( 
So I'd vote for keeping default to false, and use writeProperty/readProperty as it should be. It's more symmetric. 
From a user perspective, it changes nothing since tempo from palette or via Create > Text > Tempo are created with the followtext. The only thing, it's not the default so reset is enabled. Not a problem to me.